editor/editor_spin_slider.cpp

```diff
@@ -42,7 +42,7 @@ String EditorSpinSlider::get_tooltip(const Point2 &p_pos) const {
 #else
 		Key key = Key::CTRL;
 #endif
-		return TS->format_number(rtos(get_value())) + "\n\n" + vformat(TTR("Hold %s to round to integers. Hold Shift for more precise changes."), find_keycode_name(key));
+		return TS->format_number(rtos(get_value())) + "\n" + vformat(TTR("Hold %s to round to integers. \nHold Shift for more precise changes."), find_keycode_name(key));
```

Comment:
Trailing whitespace.
```suggestion
		return TS->format_number(rtos(get_value())) + "\n" + vformat(TTR("Hold %s to round to integers.\nHold Shift for more precise changes."), find_keycode_name(key));
```

I think the double newline between the value and the hint is useful to keep though.

Reply:
The number is a bit too awkwardly spaced away, for me at least...

Reply:
Fixed both now, even if I don't agree on the double new line.

Reply:
Well the fundamental problem here is that there's a number without context followed by some unrelated information.
To me when they're both next to each other like in your screenshot it just looks like somehow left a `1` typo in the code while writing the tooltip. With a clearer separation it seems at least intentional that `1` and the hint are two separate paragraphs.

The proper way to fix this is to rethink this tooltip to make it less weird. Initially it was likely just showing the value, and someone added a hint, making it awkward. Now it probably needs to prefix the raw value with a "Value: " label or similar.

Reply:
I actually had added "Value:" in a previous draft of this same PR. I could bring it back and cut the newline here.

Reply:
I still think it's clearer with a newline. These are two unrelated bits of information -> two paragraphs.

Reply:
Oops. I already had pushed an update. Before reverting back, I can show how it would look:
![image](https://user-images.githubusercontent.com/66727710/187920792-b3f7de91-c9c3-4828-b80c-b880f1bebcf6.png)